solver/src/solver/http_solver.rs

```diff
@@ -144,86 +137,87 @@ impl HttpSolver {
             .collect()
     }
 
-    fn map_orders_for_solver(&self, orders: Vec<LimitOrder>) -> HashMap<String, LimitOrder> {
+    fn map_orders_for_solver(&self, orders: Vec<LimitOrder>) -> HashMap<usize, LimitOrder> {
         orders
             .into_iter()
             .enumerate()
-            .map(|(index, order)| (index.to_string(), order))
+            .map(|(index, order)| (index, order))
             .collect()
     }
 
     async fn order_models(
```

Comment:
If `order_cost` is no longer `async` then this method doesn't have to be `async` either.

Reply:
It also doesn't need to return a result anymore either.

Reply:
In fact it looks like `prepare_model` no longer needs to be async or anything called from in it.